config.lua.dist

```diff
@@ -135,6 +135,22 @@ experienceStages = {
 	{ minlevel = 101, multiplier = 3 }
 }
 
+-- Loyalty system
+loyaltyEnabled = true
```

Comment:
You can drop this flag and let users effectively disable the loyalty system by removing all tiers.